docs/Configuring_openshift_virtualization_vm_for_dpdk.md

```diff
@@ -96,12 +96,19 @@ spec:
           threads: 2
           cores: 5
           dedicatedCpuPlacement: true
+          isolateEmulatorThread: true
 ...
 ```
 > **Notes**
 > - The sockets field should be 1, in order to make sure the CPUs are scheduled from the same NUMA Node.
 > - In this snippet example, the VM will be scheduled with 5 [hyper-threads](https://access.redhat.com/articles/7445) (or 10 CPUs).
 
+#### SMT Aligment
+On Nodes where SMT is enabled and CPU-Manager's policy is set to `full-pcpus-only`, there is a significance to the amount of CPUs requested. If the requested total CPUs is even, the isolateEmulatorThread feature requires also doing the following in order to avoid `SMT Alignment` errors by patching the HCO CR:
+  - `oc patch hyperconvergeds.hco.kubevirt.io kubevirt-hyperconverged -n openshift-cnv --type=json -p '[{"op": "replace", "path": "/spec/featureGates/alignCPUs", "value": true}]'`
+
+More information in [BZ2228103](https://bugzilla.redhat.com/show_bug.cgi?id=2228103)
```

Comment:
The BZ was migrated to Jira ticket https://issues.redhat.com/browse/CNV-31584.
Please also change it in the PR description.

Reply:
DONE